addons/binding/org.openhab.binding.nest/ESH-INF/thing/smoke-detector.xml

```diff
@@ -3,30 +3,30 @@
 	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
-	<thing-type id="smoke_detector" listed="false">
+	<thing-type id="smoke_detector">
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="account" />
 		</supported-bridge-type-refs>
 
 		<label>Nest Protect</label>
-		<description>The smoke detector/Nest Protect for the account.</description>
+		<description>The smoke detector/Nest Protect for the account</description>
 
 		<channels>
 			<channel id="ui_color_state" typeId="UiColorState">
 				<label>UI Color</label>
 				<description>The current color of the ring on the smoke detector</description>
 			</channel>
 			<channel id="low_battery" typeId="system.low-battery">
-				<label>Battery</label>
-				<description>The state of the battery on the nest protect (if it is battery powered)</description>
+				<label>Battery Low</label>
```

Comment:
I'd remove label & description, because they should be used from the system channel type definition.

Reply:
Gone as well.